ReText/editor.py

```diff
@@ -401,6 +401,23 @@ def updateTextStatistics(self):
 			wordCount += 1
 		self.statistics = (wordCount, alphaNumCount, characterCount)
 
+	def wheelEvent(self, e: QWheelEvent):
+		modifiers = QGuiApplication.keyboardModifiers()
+		if modifiers == Qt.ControlModifier:
+			font = globalSettings.editorFont
+			size = font.pointSize()
+			scroll = e.angleDelta().y()
+			if (scroll > 0):
+				size = size+1
+			elif (scroll < 0):
+				size = size-1
```

Comment:
`size += 1` / `size -= 1` would be a bit cleaner.

Also no parentheses are needed around `scroll > 0` / `scroll < 0` in Python (unlike C-like languages).

Reply:
Yep, habits from other languages :/